src/main/scala/za/co/absa/hyperdrive/trigger/api/rest/controllers/RestErrorHandler.scala

```diff
@@ -38,6 +39,12 @@ class RestErrorHandler {
     new ResponseEntity(HttpStatus.BAD_REQUEST)
   }
 
+  @ExceptionHandler(Array(classOf[AccessDeniedException]))
+  def handleAccessDeniedException(ex: AccessDeniedException, request: WebRequest): ResponseEntity[Object] = {
+    logger.error("Access is denied")
```

Comment:
Did you not pass the `ex` as a 2nd argument on purpose?

Reply:
Yes, but I have no argument for it.
I added it to log.